srcpkgs/runit-void/files/os-release

```diff
@@ -1,4 +1,4 @@
-NAME="void"
-ID="void"
-DISTRIB_ID="void"
-PRETTY_NAME="void"
+NAME="VoidLinux"
```

Comment:
Should it be `Void Linux`?
The name in our website have the space inside it,
and our GitHub organisation also has a dash (space substitution?) inside the name.
Ditto for `PRETTY_NAME`

Reply:
Well, I first changed it to a single word but you are right, `NAME` and `PRETTY_NAME` can contain spaces. I changed it. Let's wait and see what @void-linux/pkg-committers think about this PR in general and the name(s) in detail.

Reply:
This will break ansible OS detection as written.  I need to make sure that the void support in ansible is going to be able to match both before this gets merged.

Reply:
I think we can as well leave `NAME="void"` because `PRETTY_NAME` is what a user gets to see, if it is defined.

Reply:
This will also break salt-stack.
Can we not change this string?

Reply:
@Vaelatern done